ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/DefaultTestBase.java

```diff
@@ -13,14 +13,16 @@
 package tech.pegasys.ethsigner.jsonrpcproxy;
 
 import java.io.IOException;
+import java.util.concurrent.ExecutionException;
 
 import org.junit.jupiter.api.BeforeAll;
 import org.web3j.crypto.CipherException;
 
 public class DefaultTestBase extends IntegrationTestBase {
   @SuppressWarnings("unused")
   @BeforeAll
-  private static void setupEthSigner() throws IOException, CipherException {
+  private static void setupEthSigner()
```

Comment:
Any benefit in throwing multiple exceptions? Being part of the test DSL we are never going to catch them separately they just going to end up causing the test to fail if any of them occur. So thinking of replacing them with a single Exception?

Reply:
yea, I can do that.